src/modules/getBullsAndCows.js

```
@@ -13,7 +13,39 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const res = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  const userInputArr = userInput.toString().split('');
+  const numberToGuessArr = numberToGuess.toString().split('');
```

Consider normalizing and validating input immediately after conversion to arrays. Right now the function calls `userInput.toString().split('')` and `numberToGuess.toString().split('')` (creating `userInputArr` and `numberToGuessArr`) but assumes both are 4-length arrays. Add explicit normalization and/or length checks to fail early if the function is called with unexpected input (e.g., `const a = String(userInput).trim(); const b = String(numberToGuess).trim(); if (a.length !== 4 || b.length !== 4) { throw new Error('Expected 4-digit inputs'); }`). This makes the function safer when used directly.